src/mocks/gjelderfra-mock.ts

```diff
@@ -1,16 +1,20 @@
-function hentGjelderFraDato() {
-    const fallbackDato = '2022-06-30';
-    try {
-        const gjelderFraDato = new URLSearchParams(window.location.search).get('gjelderFraDato');
-        return gjelderFraDato || fallbackDato;
-    } catch (error) {
-        console.error(error);
-        return fallbackDato;
```

Comment:
Bruker hentGjelderFraDato fra demo-staten i stedet for denne funksjonen. Den andre gjør det samme, men uten fallback eller feilhåndtering, så jeg vet ikke om noe av dette burde vært beholdt. Men jeg tenker det kanskje er greit å droppe fallback når det ikke er noe feil, så får vi vist den initielle teksten som skal komme opp når dato er null?